### PR TITLE
Prep repo

### PR DESCRIPTION
**Pull request by jburel**
Replaced https://github.com/ome/omero-web-zarr/pull/4

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+
+omero-web-zarr
+==============
+
+OMERO.web plugin for OME-Zarr.
 
 Implementation of [OME-NGFF](https://ngff.openmicroscopy.org/latest/) API with an omero-web app.
 
-Currently this app only supports OME-NGFF v0.1.
+This plugins supports OME-NGFF v0.1, v03, v0.4.
```
Comment (will-moore) [inline README.rst:14]: plugin (no s)

File: README.rst [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
 For a given Image ID in OMERO, the following URL will refer to an OME-NGFF image:
 
     https://[omero-server]/zarr/image/ID.zarr/
 
-NB: This app has not been extensively testing and should not be considered "production ready".
+Note: This app has not been extensively testing and should not be considered "production ready".
```
Comment (will-moore) [inline README.rst:19]: testing -> tested

File: README.rst [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
-    # Allow to open-with Vizarr
-
-    $ omero config append omero.web.open_with '["web_zarr_vizarr", "omero_web_zarr_index", {"supported_objects":["image"], "label": "Vizarr", "script_url": "omero_web_zarr/openwith.js"}]'
+Configuration
 
-    # Open with ome-ngff-validator
+::
 
-    $ omero config append omero.web.open_with '["web_zarr_validator", "omero_web_zarr_index", {"supported_objects":["image"], "label": "NGFF validator", "script_url": "omero_web_zarr/openwith_validator.js"}]'
+    $ omero config append omero.web.apps '"omero_web_zarr"'
```
[the remark this answers is not in the document]
Comment (jburel) [reply README.rst:35]: If I remember this was not working
Is it working now?

**Review by will-moore — APPROVED** [review]
Looks good, thx 👍

**Pull request merged**